lib/ember-cli/engine.rb

```diff
@@ -4,9 +4,11 @@ class Engine < Rails::Engine
       ActionView::Base.send :include, ViewHelpers
     end
 
-    initializer "ember-cli-rails.inflector" do
-      ActiveSupport::Inflector.inflections :en do |inflect|
-        inflect.acronym "CLI"
+    if ActiveSupport::VERSION::MAJOR > 3 && ActiveSupport::VERSION::MINOR > 1
```

Comment:
Could we do something less verbose here like `Rails.version > "3.2"`?

Not sure is `Rails.version` is a thing it 3.1.*, but there probably is something.